tests/unit/runtime/half_precision/onebit/test_onebit.py

```diff
@@ -1159,20 +1124,12 @@ def test(self, topo_config):
         }
 
         topo = PipeTopo(**topo_config)
-        steps = 500  # Must be >=100
-
-        # Allocate model for consistent initial weights.
-        init_net = AlexNetPipe()
+        steps = 10
```

Comment:
Should this be 100?

Reply:
I have it at 10 because we're not doing any correctness checks here (e.g., comparing loss with a baseline), just verifying that the Pipeline runs

Reply:
Can we ensure we are escaping fp16 overflow within 10 steps? If we are not then the optimizer step won't be triggered at all

Reply:
I reverted this back to 100 steps - I just checked and it doesn't save _that_ much time with having it at only 10 steps